lib/event_serializer/schema_registry_cache.ex

```diff
@@ -95,34 +95,41 @@ defmodule EventSerializer.SchemaRegistryCache do
     ]
   """
   def fetch_schemas do
-    schema_name_id = key_schema_name() |> fetch_id() |> make_encoder()
-    schema_value_id = value_schema_name() |> fetch_id() |> make_encoder()
-
-    format_response(schema_name_id, schema_value_id)
+    topics()
+    |> Enum.flat_map(&fetch_schema/1)
   end
 
   def fetch_id(name), do: name |> schema_registry_adapter().schema_id_for()
 
-  def key_schema_name, do: topic() <> "-key"
-  def value_schema_name, do: topic() <> "-value"
+  def key_schema_name(topic), do: topic <> "-key"
+  def value_schema_name(topic), do: topic <> "-value"
+
+  defp fetch_schema(topic) do
+    schema_name_id = key_schema_name(topic) |> fetch_id() |> make_encoder()
+    schema_value_id = value_schema_name(topic) |> fetch_id() |> make_encoder()
```

Comment:
These two lines are currently failing Travis as they do not start with a raw value.

So options would be to pipe it like:
```
topic |> key_schema_name() |> fetch_id() |> make_encoder()
```

Or to extract the pipe into another function with arity of /1 and pass and chain it there like:
```
defp thing(topic) do
topic 
|> key_schema_name() 
|> fetch_id() 
|> make_encoder()
end
```

Reply:
Fixed